### ISSUE TITLE
About loss function

### ISSUE THREAD

**Issue by 1049451037** [tkipf/gcn#87]
Hi, I find that loss in gcn code is accumulating in the *_loss* function. So I am wondering why the accumulating loss will decrease while training?

```python
    def _loss(self):
        # Weight decay loss
        for var in self.layers[0].vars.values():
            self.loss += FLAGS.weight_decay * tf.nn.l2_loss(var)

        # Cross entropy error
        self.loss += masked_softmax_cross_entropy(self.outputs, self.placeholders['labels'],
                                                  self.placeholders['labels_mask'])
```

**Comment by 1049451037** [comment]
Sorry, I misunderstood the _loss function. It is called only once. I have thought it was called every backpropagation.
